lib/providers/services/firebase/push_notification_service.dart

```diff
@@ -13,43 +13,46 @@ class PushNotificationService {
 
   static final PushNotificationService _instance = PushNotificationService._();
 
-  final FirebaseMessaging _firebaseMessaging = FirebaseMessaging();
+  final FirebaseMessaging _firebaseMessaging = FirebaseMessaging.instance;
   bool _initialized = false;
   String token;
 
   Future initialise(BuildContext context) async {
-    if (!_initialized) {
-      if (Platform.isIOS) {
-        _firebaseMessaging.onIosSettingsRegistered.listen((data) {
-          print('onIosSettingsRegistered data: $data');
-        });
+    // TODO(gguij002): Trouble migrating to flutter 2, Fix in v2
```

Comment:
this is my biggest concern, we need to address this ASAP

@RaulUrtecho  @n13

Reply:
I understand the line to request permissions in IOS, but this stream had never really used it for what is it?

Reply:
this was used i think. 
we have working push notifications in IOS and Android

Reply:
Maybe the lastest version of FCM remove this xD